snyk/managers.py

```diff
@@ -396,3 +397,39 @@ def filter(self, **kwargs: Any):
         post_body = {"filters": filters}
         resp = self.client.post(path, post_body)
         return self.klass.from_dict(self._convert_reserved_words(resp.json()))
+
+
+class IssueSetAggregatedManager(SingletonManager):
+    def all(self) -> Any:
+        return self.filter()
+
+    def filter(self, **kwargs: Any):
+        path = "org/%s/project/%s/aggregated-issues" % (
+            self.instance.organization.id,
+            self.instance.id,
+        )
+        default_filters = {
```

Comment:
Hi @nirfuchs, 

Thanks for the feedback.

if I am not mistaken, the only values that can be provided for this two fields are "True" or "False". If we set here a default value, we'll be filtering the results by default. For me, as a pysnyk user, this is a bit misleading if you call the "all()" method and by default the response is being filtered. I thought I would be more "clear" if we let the user filter explicitly if needed.
What do you think?

Reply:
Hi @nirfuchs ,

Thanks for the feedback.

if I am not mistaken, the only values that can be provided for this two fields are "True" or "False". If we set here a default value, we'll be filtering the results by default. For me, as a pysnyk user, this is a bit misleading if you call the "all()" method and by default the response is being filtered. I thought I would be more "clear" if we let the user filter explicitly if needed.
What do you think?

Reply:
Good call, @rmoldesc, but wouldn't that lead to different behavior of `aggregated issues` and `issues` objects?

pysnyk aims to replicate the Snyk API, therefore, adheres to the logic of the issues method.

Reply:
Exactly @nirfuchs, from my point of view I think that the `issues` object should't be filtering the results by default neither. I didn't propose any changes on it because is out of the scope of this PR.

Anyway, if you do not consider this to be OK I can change the implementation.

Regards.

Reply:
👍